tests/json/schema-1-tamps/input.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import psi4
+import json
+
+#Generate JSON data
+
+method = "CCSD"
+basis = "sto-3g"
+json_data = {
+    "schema_name": "qcschema_input",
+    "schema_version": 1,
```

Comment:
this key is deprecated, see `AtomicInput` in `qcel.models.results` to see the current verified QCSchema keywords.